flake.nix

```diff
@@ -7,7 +7,7 @@
     };
   };
 
-  outputs = inputs: inputs.nixCargoIntegration.lib.makeOutputs {
+  outputs = inputs@{nixpkgs, nixCargoIntegration, ...}: inputs.nixCargoIntegration.lib.makeOutputs {
```

Comment:
```suggestion
  outputs = inputs@{nixpkgs, nixCargoIntegration, ...}: nixCargoIntegration.lib.makeOutputs {
```

Reply:
oops! I added that in for testing and forgot to take it out